lib/pg_sequencer/schema_dumper.rb

```diff
@@ -22,7 +22,7 @@ module SchemaDumper
     extend ActiveSupport::Concern
 
     included do
-      alias_method_chain :tables, :sequences
+      alias_method :tables, :sequences
```

Comment:
when I updated service from rails 4.2.7.1 to 5.1.0 and ran specs, rails wrote:
NoMethodError:
  undefined method `alias_method_chain' for ActiveRecord::SchemaDumper:Class
  Did you mean?  alias_method

by this reason I added these changes into this gem